airbyte-integrations/connectors/destination-databricks/src/main/resources/spec.json

```diff
@@ -0,0 +1,118 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/databricks",
+  "supportsIncremental": true,
+  "supportsNormalization": false,
+  "supportsDBT": false,
+  "supported_destination_sync_modes": ["overwrite", "append"],
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Databricks Destination Spec",
+    "type": "object",
+    "required": [
+      "databricks_server_hostname",
+      "databricks_http_path",
+      "databricks_personal_access_token",
+      "s3_bucket_name",
+      "s3_bucket_path",
+      "s3_bucket_region",
+      "s3_access_key_id",
+      "s3_secret_access_key"
+    ],
+    "additionalProperties": false,
+    "properties": {
+      "databricks_server_hostname": {
+        "title": "Databricks Cluster Server Hostname",
+        "type": "string",
+        "description": "",
+        "examples": ["abc-12345678-wxyz.cloud.databricks.com"]
+      },
+      "databricks_http_path": {
+        "title": "Databricks Cluster HTTP Path",
+        "type": "string",
+        "description": "",
+        "examples": ["sql/protocolvx/o/1234567489/0000-1111111-abcd90"]
+      },
+      "databricks_port": {
+        "title": "Databricks Cluster Port",
+        "type": "string",
+        "description": "",
+        "default": "443",
+        "examples": ["443"]
+      },
+      "databricks_personal_access_token": {
+        "title": "Databricks Personal Access Token",
+        "type": "string",
+        "description": "",
+        "examples": [""],
+        "airbyte_secret": true
+      },
+      "database_schema": {
+        "title": "Database Schema",
+        "type": "string",
+        "description": "The default schema tables are written to if the source does not specify a namespace. Unless specifically configured, the usual value for this field is \"public\".",
+        "default": "public",
+        "examples": ["public"]
+      },
+      "s3_bucket_name": {
```

Comment:
cloud-specific configuration should probably go in a `oneOf` block so we can seamlessly support more cloud providers in the future.

Reply:
Good point. Will do.